### PR TITLE
add funciton for dpr burn for ezc

### PR DESCRIPTION
**Pull request by jerry-yu**
add funcito for dpr burn for ezc

### CODE REVIEW

File: pallets/operation/src/lib.rs [hunk 1]
```diff
@@ -341,6 +349,25 @@ pub mod pallet {
...
+            burned: BalanceOf<T>,
+            benifity: H160,
+        ) -> DispatchResultWithPostInfo {
+            let sender = ensure_signed(origin)?;
+
+            if T::Currency::can_slash(&sender, burned) {
+                let (burned, _) = T::Currency::slash(&sender, burned);
+                let balance = burned.peek();
+                T::BurnedTo::on_unbalanced(burned);
+                Self::deposit_event(Event::<T>::BurnForEZC(sender, balance, benifity));
```
Comment (xcaptain) [inline pallets/operation/src/lib.rs:365]: Is `balance` the same as `burned`?

File: pallets/operation/src/lib.rs [hunk 2]
```diff
@@ -341,6 +349,25 @@ pub mod pallet {
             }
             Ok(().into())
         }
+
+        #[pallet::weight(T::OPWeightInfo::burn_for_ezc())]
+        pub fn burn_for_ezc(
```
Comment (xcaptain) [inline pallets/operation/src/lib.rs:354]: MIssing unit test

File: pallets/operation/src/lib.rs (hunk 1 above)
Comment (jerry-yu) [reply pallets/operation/src/lib.rs:365]: ?

Comment (xcaptain) [reply pallets/operation/src/lib.rs:365]: `let balance = burned.peek();` is `balance` actually burned DPR amount?

Comment (jerry-yu) [reply pallets/operation/src/lib.rs:365]: Yes，maybe using withdraw is more suitable

**Review by xcaptain — APPROVED** [review]

File: pallets/operation/src/lib.rs [hunk 3]
```diff
@@ -341,6 +348,26 @@ pub mod pallet {
...
+            origin: OriginFor<T>,
+            burned: BalanceOf<T>,
+            benifity: H160,
+        ) -> DispatchResultWithPostInfo {
+            let sender = ensure_signed(origin)?;
+
+            let burned = T::Currency::withdraw(
```
[the remark this answers is not in the document]
Comment (jerry-yu) [reply pallets/operation/src/lib.rs:360]: ezc also has decmals, maybe 18

File: pallets/operation/src/tests.rs [hunk 4]
```diff
@@ -206,3 +207,12 @@ fn set_staking_release_info() {
         assert_eq!(Operation::accounts_release_info(&4).is_none(), true);
     });
 }
+
+#[test]
+fn burn_for_ezc() {
+    new_test_ext().execute_with(|| {
+        assert_eq!(Balances::free_balance(&1), 98);
+        assert_ok!(Operation::burn_for_ezc(Origin::signed(1), 48, H160::zero()));
+        assert_eq!(Balances::free_balance(&1), 50);
```
[the remark this answers is not in the document]
Comment (jerry-yu) [reply pallets/operation/src/tests.rs:216]: it is so simple funciton

**Review by xcaptain — APPROVED** [review]

**Pull request merged**